### ISSUE TITLE
The system cannot find the file specified

### ISSUE THREAD

**Issue by vhphuc** [rodydavis/flutter_scripts#1]
Command line throw following error on Win 11. Command run in the project root path

Error

```
Running: freeze -> flutter pub run build_runner build --delete-conflicting-outputs

Unhandled exception:
ProcessException: The system cannot find the file specified.
```

My pubspec.yaml

    scripts:
      laucher_icons: flutter pub run flutter_launcher_icons:main
      freeze: flutter pub run build_runner build --delete-conflicting-outputs
      build_android: flutter build appbundle -t lib/src/main_release.dart
      build_ios: flutter build ios --no-codesign -t lib/src/main_release.dart

**Comment by rodydavis** [comment]
Which platform are you running on?

**Comment by vhphuc** [comment]
I run it on Window 11, Powershell. Tried on 2 PC.
I installed with this cmd
```
dart pub global activate flutter_scripts
```

**Comment by rodydavis** [comment]
Haven’t tested on windows, but I can try a few things!

**Comment by TahaTesser** [comment]
Running into this issue on my personal machine (Windows 11).

**Comment by rodydavis** [comment]
Interesting. Does windows handle dart Process differently for where the script is run from?

**Comment by pankaj-nikam** [comment]
Yes, same issue on Windows 11. I am not sure if the root cause is Windows 😉

**Comment by rodydavis** [comment]
I have a windows test device now and will work on getting it to behave the same!

**Comment by S0NN1** [comment]
Any updates on the matter?

**Comment by swarajkumarsingh** [comment]
Hey everyone, it is working on windows 10, but not sure about windows 11

**Comment by ChristophKe** [comment]
@swarajkumarsingh  still running in this problemon windows 10.

**Comment by swarajkumarsingh** [comment]
> @swarajkumarsingh still running in this problemon windows 10.

Could you please share the screenshot and flutter doctor results

**Comment by swarajkumarsingh** [comment]
> @swarajkumarsingh still running in this problemon windows 10.

Yes it is still working

**Comment by rodydavis** [comment]
I'll work on cutting a new release and port it to the latest tool chain
